packages/app/components/item/DocumentPicker/DocumentPickerNative.tsx

```diff
@@ -0,0 +1,2 @@
+import DocumentPicker from 'react-native-document-picker';
```

Comment:
im fine with this but we might want to consider https://docs.expo.dev/versions/latest/sdk/document-picker/ just so we don't have to split web and native up

Reply:
Ok, good.